syntax = "proto3";

package FKVerifyGrpc;

service VarifyCodeService {
  rpc GetVarifyCode (VarifyCodeRequestBody) returns (VarifyCodeResponseBody) {}
}

message VarifyCodeRequestBody {
  string email = 1;
}

message VarifyCodeResponseBody {
  int32 error = 1;
  string email = 2;
  string code = 3;
}